src/content/self-hosted/helm-configuration.mdx

```diff
@@ -668,6 +668,7 @@ These annotations and labels are additional to the ones already applied by Oktet
 - `labels`: Labels applied to each namespace created by Okteto.
 - `ingress.annotations`: Annotations applied to ingresses in namespaces created by Okteto.
 - `ingress.labels`: Labels applied to ingresses in namespaces created by Okteto.
+- `autoRoleBinding.enabled`: Flag to automatically bind `default` service account within namespaces managed by Okteto to the cluster role specified on `serviceAccounts.roleBindings.namespaces`. Default to `true`.
```

Comment:
I would mention that this gives access to the Kubernetes API for all pods without a custom service account

Reply:
Done at https://github.com/okteto/docs/pull/768/commits/1600fb78f1dc1fc93a1b4689c2d414dbb880ad78